engines/libaio.c

```diff
@@ -13,13 +13,28 @@
 #include "../lib/pow2.h"
 #include "../optgroup.h"
 
+// Should be defined in newest aio_abi.h
```

Comment:
We prefer C style comments rather than C++ ones where possible.

Reply:
Ok, I'll convert the comments wherever I've made them